elastic/kibana
# [Response Ops][Alerting] Skip alerts resource installation on `migrator` nodes

Resolves https://github.com/elastic/response-ops-team/issues/148

## Summary

This PR makes the following changes:
* Skips the initialization of the `AlertsService` for migrator nodes. This bypasses the installation of alerts as data assets on migrator nodes, which are short-lived nodes used on Serverless
* Changes error logs to debug logs when alert resource installation is cut short due to Kibana shutdown.

## To Verify

Run Kibana with the following configs:
* no config for `node.roles` - Kibana should start up and install alerts-as-data resources
* `node.roles: ["background_tasks"]` - Kibana should start up and install alerts-as-data resources
* `node.roles: ["ui"]` - Kibana should start up and install alerts-as-data resources
* `node.roles: ["migrator"]` - Kibana should start up and not install alerts-as-data resources. No errors related to premature shutdown should be visible in the logs.
